src/tree_gui.cpp

```diff
@@ -133,7 +133,7 @@ class BuildTreesWindow : public Window
 		}
 		const uint radius = this->mode == PM_FOREST_LG ? 12 : 5;
 		const uint count = this->mode == PM_FOREST_LG ? 12 : 5;
-		PlaceTreeGroupAroundTile(tile, treetype, radius, count);
+		PlaceTreeGroupAroundTile(tile, treetype, radius, count, this->tree_to_plant != TREE_INVALID);
```

Comment:
I don't think I follow why the `set_zone` parameter is necessary at all? Under what circumstances would the tree_to_plant be invalid (so picking a random type), but you *wouldn't* want to set the tropic zone?

Reply:
If you plant random trees with the single-tile brush, it doesn't create rainforest either.

Reply:
Can we get a comment explaining that?